### PR TITLE
Task#7500 

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
created components for taxonomy terms. Did some changes in views.

### CODE REVIEW

**Comment by ivandudardb** [comment]
LGTM

File: web/themes/custom/my_awesome_theme/assets/images/38.jpg
Comment (lavrenchuk-dev) [inline web/themes/custom/my_awesome_theme/assets/images/38.jpg:1]: This file name doesn't describe anything about where and why it is used.

File: web/themes/custom/my_awesome_theme/components/03-organisms/site/site-gadgets-page/_site-gadgets-page.scss [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                  white-space: nowrap;
+
+            }
+          }
+        }
+      }
+    }
+  }
+}
+**/
```
Comment (lavrenchuk-dev) [inline web/themes/custom/my_awesome_theme/components/03-organisms/site/site-gadgets-page/_site-gadgets-page.scss:97]: Please stop with commented code

It's not hard to write "@todo Can be useful later" or anything
And if you have literally nothing to write in @todo than it is useless and shouldn't be in the repo.

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

File: web/themes/custom/my_awesome_theme/assets/images/38.jpg
Comment (HeorhiiBoliuk) [reply web/themes/custom/my_awesome_theme/assets/images/38.jpg:1]: This file while not using

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**